### ISSUE TITLE
Checkfront Integration

### ISSUE THREAD

**Issue by AlonsoIndex18** [PipedreamHQ/pipedream#5547]
We would like a checkfront integration for triggers. 

https://api.checkfront.com/ This is the API documentation

There is a limitation because Checkfront only allows the users to have 1 webhook at the same time. It would be easier to build if we have native triggers in pipedream for checkfront and save the webhooks for other uses.

We would like to see triggers when a booking state is changed and when a payment or refund is applied.

**Comment by sergioeliot2039** [comment]
Checkfront has OAuth integratino method:
https://api.checkfront.com/overview/auth.html#authenticating-with-oauth2

**Comment by sergioeliot2039** [comment]
**App**
Checkfront
https://api.checkfront.com/

**Triggers**
Booking State Changed
Payment/Refund Applied

webhooks available at: https://api.checkfront.com/ref/notifications.html

**Comment by AlonsoIndex18** [comment]
Yes, we are already using that method for authentication. We are also using the Checkfront webhooks too. But the problem is that you can only maintain one webhook at the time so you can't use it for multiple applications.

**Comment by sergioeliot2039** [comment]
Hi @AlonsoIndex18 , thanks for clarifying.

I specified the OAuth method for the Pipedream team to know this must be used when building the apps.
As of webhooks, are you suggesting that triggers/source components to be built on Pipedream should be done by polling the API and not webhooks becuse of the limitation?

**Comment by AlonsoIndex18** [comment]
Yes, the trigger for the Pipedream integration should be done polling the Checkfront API!

**Comment by sergioeliot2039** [comment]
All, thanks for the explanation. I have updated the requirement about the triggers.

Keep an eye on this issue, it will be updated soon by the component development team.

**Comment by lcaresia** [comment]
I'm looking for this task and don't found the app.

Cc: @dannyroosevelt

**Comment by sergio-eliot-rodriguez** [comment]
They reach back from their partnership program application.

**Comment by sergio-eliot-rodriguez** [comment]
I followed up with their partnership team.
